script/validators/source_validator.py

```diff
@@ -219,8 +223,31 @@ def check_namespaces(file_path):
     return status
 
 
+def check_includes(file_path):
+    """Checks whether local includes are done via #include<...>"""
+    with open(file_path, "r") as f:
+        path_pattern = re.compile(r'^#include <(.*?)>')
+        includes = os.listdir('src/include')
+        linenum = 0
+        status = True
+        for line in f:
+            linenum += 1
+            res = path_pattern.match(line)
+            if res:
+                path = res.groups()[0].lstrip("include/")
```

Comment:
`lstrip` also doesn't work here. But I just realized you can deal with this also in the regex itself: `^#include <(include\/)?(.*?)>` (and then take the second match group)